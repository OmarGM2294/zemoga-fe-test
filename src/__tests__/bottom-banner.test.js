import React from 'react'

import { create } from 'react-test-renderer'

import BottomBanner from '../components/bottom-banner'

describe('Bottom Banner', () => {
  it('Should match snapshot (it renders correctly)', () => {
    const component = create(<BottomBanner />)

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
