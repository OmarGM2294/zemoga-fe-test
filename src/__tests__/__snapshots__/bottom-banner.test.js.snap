// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottom Banner Should match snapshot (it renders correctly) 1`] = `
<aside>
  <h2>
    Is there anyone else you would want us to add?
  </h2>
  <button
    type="button"
  >
    Submit a name
  </button>
</aside>
`;
